x-pack/plugins/alerting/docs/openapi/bundled.json

```diff
@@ -3064,30 +3091,42 @@
       "update_rule_request": {
         "summary": "Update an index threshold rule.",
         "value": {
-          "notify_when": "onActionGroupChange",
+          "actions": [
+            {
+              "frequency": {
+                "summary": false,
+                "notify_when": "onActionGroupChange"
```

Comment:
We need `throttle` here. 
if it's not set, the value would be `null`

Reply:
Right, this is just an example for the update rule endpoint and seemed to work when I tested it locally.